src/UITests/LoadEventFile.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.IO;
+using System.Reflection;
+
+namespace UITests
+{
+    /// <summary>
+    /// Start the app, open an A11yEvent file, spot check loaded data, exit app
+    /// </summary>
+    [TestClass]
+    public class LoadEventFile : AIWinSession
+    {
+        static readonly string EventFileName = "WildlifeManagerTest.a11yevent";
+        static readonly string TestFilePath = $"{Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location)}\\TestFiles";
+
+        /// <summary>
+        /// The entry point for this test scenario. Every TestMethod  will restart ai-win, so
+        /// we want to use them sparingly.
+        /// </summary>
+        [TestMethod]
+        [TestCategory("NoStrongName")]
+        [TestCategory("UITest")]
+        public void LoadEventFileTests()
+        {
+            driver.Maximize();
```

Comment:
Do we need this to make sure clicking happens correctly across different screens?

Reply:
This is just to make it easier to see everything because the screen is kind of cramped - helps with the video